### DISCUSSION TITLE
Error while installing Foundry

### DISCUSSION THREAD

**Discussion by 0xHuzaifa** [foundry-rs/foundry#6915] [Help]
Hello everyone, I'm getting an error while installing Foundry using this command

**Command:**
```
curl -L https://foundry.paradigm.xyz | bash
```

**Error:**
```
 ~/desktop/foundry
$ curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  1942  100  1942    0     0   1318      0  0:00:01  0:00:01 --:--:--  1318Installing foundryup...

Warning: Failed to open the file C:/Users/Huzaifa: Permission denied
####################                                                      28.8%curl: (23) Failure writing output to destination

curl: (6) Could not resolve host: Ahmed
```

I'm using **GitBash** normally and as an **Administrator**. But getting the same error. 
I tried to install it using Visual Code but nothing changed.

**Comment by Praveen2645** [comment]
do you got any solution for this, I got the same issue

**Reply by 0xHuzaifa** [reply]
Yes,

I get this error because there is a gap in my Windows user name. "Huzaifa Ahmed".

So I install WSL ubuntu to install foundry. and it worked for me.

**Comment by DaniPopes** [comment]
Should be fixed with https://github.com/foundry-rs/foundry/pull/7168

**Reply by Rick366** [reply]
I can't understand the specific proccess,can u express it.Thanks!
